CONTRIBUTING.md

```diff
@@ -4,5 +4,5 @@ To help you get started on the right track, check out the following resources to
 
 - The [Article Style Guide](https://new.ppy.sh/wiki/Article_Style_Guide) for the styling standards in wiki.
 
-- The [Getting Started Guide](https://new.ppy.sh/wiki/osu!wiki_contribution_guide) _(New to the GitHub workflow?, you should read this first)_.
+- The [Getting Started Guide](https://new.ppy.sh/wiki/osu!wiki_contribution_guide) _(New to the GitHub workflow? You should read this first)_.

```

Comment:
I'd use new.ppy.sh just for styles (and for the convenience of it selecting the locale for us).
Since the listed article is targeting the user's language, new.ppy.sh should redirect for us (instead of having the person navigate through the repo).